tsconfig.json

```diff
@@ -12,8 +12,8 @@
       "backburner": ["lib/index.ts"]
     }
   },
-  "files": [
+  "file": [
```

Comment:
I don't grok this change, I didn't think `file` was an option for tsconfig.json?